sites/site-utilities/src/definitions/native/html-native.definition.ts

```diff
@@ -0,0 +1,80 @@
+import { DataType } from "@microsoft/fast-tooling";
+import { WebComponentAttribute } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { voidElements } from "@microsoft/fast-tooling/dist/data-utilities/html-element";
+
+/**
+ * These definitions are dependent on a devDependency on "vscode-web-custom-data"
+ * speficially the browsers.html-data.json which defines native html elements and their attributes
+ * as used by vscode
+ */
+import * as vcodeHTMLData from "vscode-web-custom-data/data/browsers.html-data.json";
+
+const valueSets: any[] = vcodeHTMLData.valueSets;
+
+export const htmlNativeDefinitions: any = {
+    version: 1, // WebComponentDefinition has this typed as 1, vcodeHTMLData.version could be anything
+    tags: vcodeHTMLData.tags,
+};
+
+function getDataTypeForAttribute(attribute: any): DataType {
```

Comment:
Similarly here, can you create an interface for the attribute? This will help us in the future, you can just create a `html-native.vs-code-v1.1-types.ts` file to store these in.